src/components/templates/Search/index.tsx

```diff
@@ -7,6 +7,43 @@ import queryString from 'query-string'
 import { getResults } from './utils'
 import Loader from '../../atoms/Loader'
 import { useOcean } from '@oceanprotocol/react'
+import { DragDropContext, Droppable, Draggable } from 'react-beautiful-dnd'
+const defaultSearchItems = () =>
+  ['created', 'price.ocean', 'price.value'].map((e) => ({
+    id: e,
+    direction: -1
+  }))
+
+// a little function to help us with reordering the result
+const reorder = (list, startIndex, endIndex) => {
+  const result = Array.from(list)
+  const [removed] = result.splice(startIndex, 1)
+  result.splice(endIndex, 0, removed)
+
+  return result
+}
+
+const grid = 8
+
+const getItemStyle = (isDragging, draggableStyle) => ({
+  // some basic styles to make the items look a bit nicer
+  userSelect: 'none',
```

Comment:
all styling should happen in CSS module files, with switching classes based in js variables in here, like `isDragging`. But don't think we should have any drag & drop thingy here